### PR TITLE
Update monitoring docs

### LINKED ISSUE #63: Make monitoring doc consistent across Besu and GoQuorum
It's not clear which monitoring options are the same in Besu and GoQuorum and which are different.

### LINKED ISSUE #82: Update metrics content
A GoQuorum Grafana dashboard is now available here: https://grafana.com/grafana/dashboards/14360

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull Request Description

- Edit and update monitoring content
- Collect and truncate content from Concept, Reference, and Quickstart sections
- Make content slightly more consistent with Besu monitoring docs (simultaneously updated in https://github.com/hyperledger/besu-docs/pull/768)

## Fixed Issue(s)

fixes #63
fixes #82

### CODE REVIEW

File: docs/HowTo/Monitor/Elastic-Stack.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-using ELK for log management.
-
-The [Filebeat] configuration ingests logs, and the [Metricbeat] configuration collects metrics from the nodes at regular
+The [Filebeat] configuration ingests logs and the [Metricbeat] configuration collects metrics from the nodes at regular
 defined intervals and outputs them to Redis for storage.
-Redis provides a highly available storage mechanism used by Elastic Beats, and pulled by Logstash as required.
+Redis provides a highly available mechanism enabling storage by any of the Elastic Beats and pulled by Logstash as required.
 
-The [pipeline configuration] defines the JSON format used for GoQuorum logs and automatically picks up any new log fields.
+The [pipeline configuration] defines the JSON format used for Besu logs and automatically picks up any new log fields.
```
Comment (macfarla) [inline docs/HowTo/Monitor/Elastic-Stack.md:14]: why Besu? shouldn't this be GoQuorum?

**Review by macfarla — APPROVED** [review]
LGTM. one spot where Besu should be GoQuorum

File: docs/HowTo/Monitor/Cakeshop.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# Cakeshop
+
+[Cakeshop](https://github.com/ConsenSys/cakeshop) is a set of tools and APIs for working with Ethereum,
+packaged as a Java web application archive (WAR) that gets you up and running quickly.
+
+Out of the box you get:
+
+* **Blockchain explorer** - View transactions, blocks, and contracts, and see historical contract state at a point in time.
+* **Contract registry** - Keep track of deployed contracts and their code, interfaces, and state.
```
Comment (ricardolyn) [inline docs/HowTo/Monitor/Cakeshop.md:13]: note: the contracts are registered only if those are deployed through the Solidity sandbox. if you deploy by other means (using an external Remix IDE or manually through Truffle or other) won't show up there. 

not sure if is worth mentioning this.

**Review by ricardolyn — APPROVED** [review]
LGTM

**Review by bgravenorst — APPROVED** [review]
LGTM

**Pull request merged**